### PR TITLE
athena: version endpoint

### PR DESCRIPTION
**Pull request by gregjhogan**

### CODE REVIEW

File: selfdrive/athena/athenad.py [hunk 1]
```diff
@@ -132,6 +132,16 @@ def getMessage(service=None, timeout=1000):
   return ret.to_dict()
 
 
+@dispatcher.add_method
+def getVersion():
+  return {
+    "version": version,
```
Comment (pd0wm) [inline selfdrive/athena/athenad.py:138]: add get_version method for consistency?

Comment (gregjhogan) [reply selfdrive/athena/athenad.py]: good idea - done

File: selfdrive/athena/tests/test_athenad.py [hunk 2]
```diff
@@ -174,6 +174,13 @@ def test_getSshAuthorizedKeys(self):
     keys = dispatcher["getSshAuthorizedKeys"]()
     self.assertEqual(keys, MockParams().params["GithubSshKeys"].decode('utf-8'))
 
+  def test_getVersion(self):
+    resp = dispatcher["getVersion"]()
+    keys = ["version", "remote", "branch", "commit"]
+    self.assertEqual(list(resp.keys()), keys)
+    for k in keys:
+      self.assertTrue(resp[k], f"{k} has no value")
```
Comment (adeebshihadeh) [inline selfdrive/athena/tests/test_athenad.py:182]: check type instead?

Comment (gregjhogan) [reply selfdrive/athena/tests/test_athenad.py:182]: well I want an error if it returns None or an empty string (the default if an error happens), so this seems better than just checking if it is a string?

Comment (adeebshihadeh) [reply selfdrive/athena/tests/test_athenad.py:182]: can we check those explicitly? it's a bit unclear from the way it's written
```suggestion
      self.assertTrue(isinstance(resp[k], str) and len(k), f"{k} has no value")
```

Comment (gregjhogan) [reply selfdrive/athena/tests/test_athenad.py:182]: yeah, sounds good!

**Pull request merged**